Baby Monitor/Resources/Constants.swift

```diff
@@ -15,25 +15,25 @@ enum Constants {
     static let cryingDetectionThreshold = 0.7
 
     /// A limit per which one notification about cry detection can be sent.
-    static let cryingNotificationRequestTimeLimit: TimeInterval = 3 * 60
+    static let cryingNotificationRequestTimeLimit: Int = 3 * 60
 
     /// A limit per which noise detection notification can be sent.
-    static let noiseNotificationRequestTimeLimit: TimeInterval = 15
+    static let noiseNotificationRequestTimeLimit: Int = 15
 
     /// A time after which the video stream should be hidden.
-    static let videoStreamVisibilityTimeLimit: TimeInterval = 60
+    static let videoStreamVisibilityTimeLimit: Int = 60
 
     /// A limit for searching for a baby device to be paired to.
     static let pairingDeviceSearchTimeLimit: TimeInterval = 2 * 60
 
     /// A limit for sending maximum one sound to an ML model for recognizing in this time.
-    static let recognizingSoundTimeLimit: TimeInterval = 10
+    static let recognizingSoundTimeLimit: Int = 10
 
     /// An initial limit for sound loudness that should be cut off.
     static let loudnessFactorLimit = 35
 
     /// A limit for which should get a confirmation id back.
-    static let webSocketConfimationIDTimeLimit: TimeInterval = 5
+    static let webSocketConfimationIDTimeLimit: Int = 5
```

Comment:
Why did you decided to change type of those constants? What do you think about adding units to their name or at least to their descriptions?

Reply:
Good point!
Changed because of the migration to RxSwift 5 and change of API.